src/cronjobs

```diff
@@ -2,6 +2,15 @@ p({"class":"foam.nanos.cron.Cron","minute":1,"hour":-1,"dayOfMonth":-1,"month":-
 p({"class":"foam.nanos.cron.Cron","id":"PMInfoRollup","scheduledTime":"2019-04-28T05:39:00.000Z","lastRun":"2019-04-28T05:38:00.802Z","code":"import foam.dao.DAO;\nimport foam.nanos.logger.Logger;\nimport foam.nanos.pm.PM;\nimport foam.nanos.pm.PMInfo;\nimport foam.nanos.pm.DAOPMLoggerCopySink;\nimport foam.nanos.script.Script;\n\nscriptName = \"PMInfoRollup\";\npm = new PM(Script.class, \"scriptName\");\nlogger = x.get(\"logger\");\nlogger.info(new String[] {scriptName, \"start\"});\n\n// current 1 minute dao\ndao = x.get(\"pmInfoDAO\").getDelegate();\n\n// replace with new empty dao\nx.get(\"pmInfoDAO\").setDelegate(new foam.dao.MDAO(foam.nanos.pm.PMInfo.getOwnClassInfo()));\n\n/*\none = x.get(\"pmInfo1minDAO\");\nfive = x.get(\"pmInfo5minDAO\");\nhour = x.get(\"pmInfo1hourDAO\");\nday = x.get(\"pmInfo1dayDAO\");\n\nlogger.info(new String[] {scriptName, \"daos\", one.toString(), five.toString(), hour.toString(), day.toString()});\nlogger.info(new String[] {scriptName, \"current replaced\"});\n\n// move previous 1 minute to named 1 minute dao\none.setDelegate(dao);\n\n// cleanup\nnow = java.time.LocalDateTime.now();\n\n// if at 5 minute interval of hour clear 5 minute dao\nif ( now.getMinute() % 5 == 0 ) {\n  five.removeAll();\n  logger.info(new String[] {scriptName,\"clean 5 minute\"});\n}  \n\n// if at top of the hour, clear hour dao, if start of day.\nif ( now.getMinute() == 0 ) {\n  hour.removeAll();\n  logger.info(new String[] {scriptName,\"clean 1 hour\"});\n  if ( now.getHour() == 0 ) {\n    day.removeAll();\n    logger.info(new String[] {scriptName,\"clean 1 day\"});\n  }\n}\n\n// put 1 minute entries into other daos.\nDAO[] daos = new DAO[] { five, hour, day };\ndao.select(new DAOPMLoggerCopySink(x, daos));\n*/\nlogger.info(new String[] {scriptName,\"end\"});\npm.log(x);\n","output":""})
-p({"class":"foam.nanos.cron.Cron","minute":0,"id":"PMHourlyReduce","enabled":false,"code":"rm = x.get(\"pmHourlyReduceManager\");\nrm.doReduce();"})
-p({"class":"foam.nanos.cron.Cron","hour":0,"id":"PMDailyReduce","enabled":false,"code":"rm = x.get(\"pmDailyReduceManager\");\nrm.doReduce();"})
-p({"class":"foam.nanos.cron.Cron","enabled":true,"id":"PmReduceManager","code":"import foam.nanos.logger.Logger;\n\nscriptName = \"PMReduceManager\";\nlogger = x.get(\"logger\");\ndailyRM = x.get(\"pmDailyReduceManager\");\nhourlyRM = x.get(\"pmHourlyReduceManager\");\nnow = java.time.LocalDateTime.now();\n\nif (now.getHour() == 0) {\n  hourlyRM.doReduce();\n  dailyRM.doReduce();\n  logger.info(new String[] {scriptName, \"Daily PM Reduce Complete\"});\n} else if (now.getMinute() == 0){\n  hourlyRM.doReduce();\n  logger.info(new String[] {scriptName, \"Hourly PM Reduce Complete\"});\n}\n"})
+p({"class":"foam.nanos.cron.Cron","minute":0,"id":"PMHourlyReduce","enabled":true,"code":"""
```

Comment:
It would be nice for readability to put all properties on their own line.

Reply:
Done